backend/inspirehep/search/facets.py

```diff
@@ -177,6 +178,41 @@ def inner(values):
     return inner
 
 
+def curation_filter(author_recid):
```

Comment:
```suggestion
def self_author_claimed_papers_filter(author_recid):
```
to be consistent with the aggregation as it's not generic at all?